core/src/test/java/io/grpc/internal/RetriableStreamTest.java

```diff
@@ -991,6 +991,34 @@ public void messageAvailable() {
     verify(masterListener).messagesAvailable(messageProducer);
   }
 
+  @Test
+  public void addPrevRetryAttemptsToRespHeaders() {
```

Comment:
Do we need another test to verify there is no `GRPC_PREVIOUS_RPC_ATTEMPTS` header on the response of the very first attempt (since the gRFC says for the value of 0 there doesn't need  to be or shound't be that header)?